File: 자연수 뒤집어 배열로 만들기.py
"""
def solution(n):
    n = list(map(int,str(n)))
    answer = []
    
    for i in range(1, len(n)+1):
        answer.append(n[-i])

    return answer
"""
#2022.11.12
def solution(n):
    return [int(v) for v in str(n)[::-1]]

File: 자연수 뒤집어 배열로 만들기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* solution(long long n) {
    // 리턴할 값은 메모리를 동적 할당해주세요.
    int* answer = (int*)malloc(4*12);
    int temp = 0;
    
    while (n > 0){
        answer[temp++] = n % 10;
        n /= 10;
    }
    return answer;
}
